util/set_meta.js

```diff
@@ -11,8 +11,7 @@ export default (meta, context) => {
     const pageDesc = `${contextDesc}${contextDesc ? ' - ' : ''}${defaultDesc}`;
 
     // Get base & page URL
-    const base = process.env.SITE_HOST || (typeof (window) !== 'undefined' ? window.location.origin : '/');
-    const pageUrl = base.trim().replace(/\/*$/, '') + context.$route.path;
+    const pageUrl = context.$base.trim().replace(/\/*$/, '') + context.$route.path;
```

Comment:
```suggestion
    const pageUrl = context.$base.slice(0, -1) + context.$route.path;
```
This can be cleaner if we know we're always returning a single trailing slash